libs/single-spa-angular/webpack/index.ts

```diff
@@ -35,16 +35,13 @@ export default (config: any, options?: Options, extraOptions?: DefaultExtraOptio
     externals: ['zone.js'],
     devServer: {
       historyApiFallback: false,
-      contentBase: path.resolve(process.cwd(), 'src'),
+      static: path.resolve(process.cwd(), 'src'),
```

Comment:
Shouldn't we use the directory options instead? 
https://webpack.js.org/configuration/dev-server/#directory

```javascript
devServer: {
    static: {
      directory: path.resolve(__dirname, "static"),
```

Reply:
I took that from your comment where you were saying that you patched single-spa config. I don't mind changing that to `static.directory`. Is there a difference between these 2 approaches? I'm asking because I'm unaware of that.

Reply:
> I took that from your comment where you were saying that you patched single-spa config

Indeed but I upgraded again and then Webpack complained again about the schema. I'll update my comment

Reply:
Comment updated with:
```javascript
  const singleSpaWebpackConfig = singleSpaAngularWebpack(config, options);

  const singleSpaWebpackConfigPatched = {
    ...singleSpaWebpackConfig,
    devServer: {
      ...singleSpaWebpackConfig.devServer,
      static: {
          directory: singleSpaWebpackConfigPatched.devServer.contentBase,
        },
      }
    },
  };
  delete singleSpaWebpackConfigPatched.devServer.contentBase;
```

but personally, I don't know why we need that?! I am in Nx repo and it is not even compatible to target the root src/...